docs/manpage_syntax.md

```diff
@@ -0,0 +1,102 @@
+% podman-command(1)
+
+## NAME
+podman\-command - short description
+
+## SYNOPSIS
+(Shows the command structure.)
+
+**podman command** [_optional_] _mandatory value_
+
+**podman subcommand command** [_optional_] _mandatory value_
+
+(If there is the possibility to chose between 2 (two) or more mandatory command values. There should also always be a space before and after a vertical bar to ensure better readability.)
+
+**podman command** [_optional_] _value1_ | _value2_
+
+**podman subcommand command** [_optional_] _value1_ | _value2_
+
+(If an optinal value follows a mandatory one.)
+
+**podman command** [_optional_] _value1_ | _value2_ [_optional_]
+
+**podman subcommand command** [_optional_] _value1_ | _value2_ [_optional_]
+
+(If the command accepts an infinite number of values.)
+
+**podman command** [_optional_] _value_ [_value_ ...]
+
+**podman subcommand command** [_optional_] _value_ [_value_ ...]
+
+## DESCRIPTION
+**podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the decription ensures uniformity. All commands mentioned in a text retain their appearance and form.\
+Example sentence: The command **podman command** is an example command.\
+It should also be specified if the command can only be run as root. In addition, it should be described when a command or OPTION cannot be executed with the remote client. For a command this should be done in the DESCRIPTION part, for the OPTIONS it should be done in the DESCRIPTION of the specified OPTION.
```

Comment:
Also, something to massage in here is to NOT use pronouns in the man pages, especially the word "you".

Reply:
I'll add that